src/core/blockchain/zilliqa/stats.ts

```diff
@@ -24,6 +25,12 @@ export class Stats extends GenericStats<Client> {
             this.client.chainId.toString()
         );
 
-        return new BigNumber(data.balance.available).toFixed();
+        let availableToDelegate = new BigNumber(data.balance.unstaked);
+        if (availableToDelegate > config.amountToKeepInAccount) {
```

Comment:
i thinik you should check this instead `data.balance.available > config.amountToKeepInAccount` also you cannot compare bignumbers like this. use lt gt functions